docs/reference/rest-api.md

```diff
@@ -114,3 +114,60 @@ The formatting is based on the specified output format.
 
 On error, an "X-Stream-Error" header will be sent via the trailers channel with information about the error, and the stream will be closed via [`sender.abort()`](https://docs.rs/hyper/0.14.16/hyper/body/struct.Sender.html#method.abort).
 Depending on the client, the trailer header with error details may not be shown. The error will also be logged in quickwit ("Error when streaming search results").
+
+
+### Ingest data in an index
+
+```
+POST api/v1/<index id>/ingest -d \
```

Comment:
Does it work with 'curl -d'? Or should we put `--data-binary`?

Reply:
Yes it's curl flag, this was a tip from @PSeitz `-d` does not load the whole file in memory.
Maybe we can covert all the verbs in curl?